docs/cookbook/recipe_improve_performance_large_datasets.rst

```diff
@@ -1,4 +1,4 @@
-Improve performance of large datasets
+Improve Performance of Large Datasets
```

Comment:
:thinking: Why?

Reply:
It changes nothing in the generated markup version, but it improves readability to me and we handle it the same way in Symfony-Docs and I think we should use their BC promise and their way of writing docs.
But I can change it back if you want

Reply:
No, I just wanted to know if it had to do with you being german or something else… and apparently it is something else ;)